packages/kit/src/core/adapt/utils.js

```diff
@@ -47,6 +48,25 @@ export function get_utils({ cwd, config, build_data, log }) {
 					log
 				});
 			}
+		},
+
+		/** @param {{patterns: string[], log?: boolean}} options */
+		update_ignores({ patterns, log = true }) {
+			const target = '.gitignore';
```

Comment:
we should also update the templates to use `.gitignore`. It looks like `eslint` might already, but not `prettier`. https://github.com/sveltejs/kit/tree/master/packages/create-svelte/shared